java/ql/src/experimental/Security/CWE/CWE-502/UnsafeSpringExporterLib.qll

```diff
@@ -0,0 +1,9 @@
+import java
+
+/**
+ * Holds if `type` is `RemoteInvocationSerializingExporter`.
+ */
+predicate isRemoteInvocationSerializingExporter(RefType type) {
+  type.getASupertype*()
+      .hasQualifiedName("org.springframework.remoting.rmi", "RemoteInvocationSerializingExporter")
```

Comment:
@artem-smotrakov just found a vulnerability on an application using the `RMIServiceExporter` and then remembered this PR. I think it would be nice to extend it so that it includes other vulnerable exporters such as the `RmiBasedExporter` or the `HessianExporter`

Reply:
Good point! Here you go https://github.com/github/codeql/pull/6142 Thanks!